protos/NUgus.proto

```diff
@@ -1283,6 +1304,20 @@ PROTO NUgus [
                         fieldOfView 1.5707
                         width 640
                         height 480
+                        spherical FALSE
+                        noise 0.000000001
+                        motionBlur 32
+                        lens Lens {
+                          center 0.5 0.5   # [0, 1] [0, 1]
+                          radialCoefficients 0 0       # any vector
+                          tangentialCoefficients 0 0       # any vector
+                        }
+                        focus Focus {
+                          focalDistance    0.0
+                          focalLength      0.00198
+                          maxFocalDistance 4.5   # [0, inf)
+                          minFocalDistance 0.1   # [0, inf)
+                        }
```

Comment:
Hmmm can probs remove camera stuff since it's being put in properly on a separate branch.